packages/@okta/vuepress-site/docs/guides/sign-users-out/define-signout-callback/aspnetcore/addbaseuri.md

```diff
@@ -1 +1,18 @@
-Follow your base URI with `/signout/callback` (for example, `http://localhost:3000/signout/callback`).
\ No newline at end of file
+Follow your base URI with `/signout/callback` (for example, `http://localhost:3000/signout/callback`).
+
```

Comment:
In ASP.NET Core you have two options to redirect to your app after Okta sign out. One approach is the same as ASP.NET (Define the sign out callback in your Developer console and adding it to your configuration via _PostLogoutRedirectUri_) or you just can add your redirect URI in the `AuthenticationProperties` as shown in line 15 of this file:

```
[HttpPost]
public IActionResult SignOut()
{
    return new SignOutResult(
        new[]
        {
                OktaDefaults.MvcAuthenticationScheme,
                CookieAuthenticationDefaults.AuthenticationScheme,
        },
        new AuthenticationProperties { RedirectUri = "/Home/" }); // Redirect to /Home after sign out
}
```